cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html

```diff
@@ -1,44 +1,61 @@
 
 {#
-  TODO: Rename file as _vars-mega-menu.html.
-
   Navigation and Sub-Navigation menu items,
-  used to populate the contents of the site's navigation menus.
-  Format: navigation title {{ string }},
-          sub-navigation title {{ string }},
-          sub-navigation media {{ HTML string }},
-          sub-navigation items {{ array of tuples }}
+  used to populate the contents of the site's navigation menus (Mega Menu).
+
+  Format:
+
+  media_items:     Array of tuples containing a media item lookup map.
+  media_items.url: The URL of the menu item.
+                   Used to look up the featured menu content.
+  media_item.fmc:  Featured Menu Content.
+
+  nav_items:                       Multi-dimensional array of tuples.
+  nav_items.nav_group:             Tuple defining groupings of nav items.
+                                   Rendered as UL HTML element.
+  nav_items.nav_group.nav_caption: Menu item link text.
+  nav_items.nav_group.nav_url:     Menu item link URL.
+  nav_items.nav_group.children:    Array of menu item submenus,
+                                   recursively repeats nav_group structure.
 #}
 
 {# TODO: Update file path for Featured Menu Content module when that is added. #}
 {% import 'templates/nav/about-us-media.html' as about_us_media %}
 
-{% set sub_nav_tbd =  ('', '', 'TBD Content') %}
-{% set nav_items = [
-    ('Consumer Tools', '#', '', '', []),
-    ('Educational Resources', '#', '', '', []),
-    ('Data & Research', '#', '', '', []),
-    ('Policy & Compliance', '#', '', '', []),
-    ('About Us', '/about-us/',  'About Us Overview', about_us_media , [
+{% set media_items = [
```

Comment:
Should these be moved out of jinja and into https://github.com/cfpb/cfgov-refresh/blob/e6a99dd9990697da950b22d76e76e4ce6dbc7065/cfgov/v1/models/ref.py? Maybe out of scope